### PR TITLE
Added option to partially hide market listings

### PR DESCRIPTION
**Pull request by ncla**
When you have many items listed on the market, and you visit the market home page, it is annoying to scroll down to other content of the page. Feature disabled by default.

### CODE REVIEW

File: BrowserExtension/scripts/community/market_sell_listings.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+        if ( element )
+        {
+            var marketOrderTypes = element.querySelectorAll( '.my_listing_section.market_content_block.market_home_listing_table' );
+
+            for ( var i = 0; i < marketOrderTypes.length; i++ )
+            {
+                var listingCount = marketOrderTypes[i].querySelectorAll( '.market_listing_row.market_recent_listing_row' ).length;
+
+                if ( listingCount > 5 )
```
Comment (xPaw) [inline BrowserExtension/scripts/community/market_sell_listings.js]: Do a `continue` here to reduce nesting

**Comment by xPaw** [comment]
This is cool, can you convert your additions to use tabs though please.

**Comment by ncla** [comment]
Think I fixed the indentation issue. Don't understand what you want me to do with the `continue` in the `for` loop. The stuff gets appended for each list there is (My buy orders and My sell listings).

**Comment by xPaw** [comment]
`if ( listingCount < 5 )
{
 continue;
}

// rest of the code`

**Comment by ncla** [comment]
Do you want those extra four lines for your own preference sake, or is there a good reason for doing that?

**Comment by xPaw** [comment]
The reason is to reduce nesting as the further code will have less indentation. Just a simple fix which you can quickly do before [squash](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/).

**Comment by ncla** [comment]
Added and squashed.

File: BrowserExtension/scripts/community/market_sell_listings.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+			{
+				var listingCount = marketOrderTypes[i].querySelectorAll( '.market_listing_row.market_recent_listing_row' ).length;
+
+				// xPaw told me so
+				if ( listingCount <= 5 )
+				{
+					continue;
+				}
+
+				if ( listingCount > 5 )
```
Comment (xPaw) [inline BrowserExtension/scripts/community/market_sell_listings.js]: Well you should have removed this `if` then

File: BrowserExtension/scripts/community/market_sell_listings.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+		if ( element )
+		{
+			var marketOrderTypes = element.querySelectorAll( '.my_listing_section.market_content_block.market_home_listing_table' );
+
+			for ( var i = 0; i < marketOrderTypes.length; i++ )
+			{
+				var listingCount = marketOrderTypes[i].querySelectorAll( '.market_listing_row.market_recent_listing_row' ).length;
+
+				// xPaw told me so
```
Comment (xPaw) [inline BrowserExtension/scripts/community/market_sell_listings.js]: Let's.. not do this :D

**Comment by ncla** [comment]
:>

**Comment by xPaw** [comment]
Are you having a giggle m8? Surely you didn't add spaces intentionally :goberserk:

**Comment by ncla** [comment]
I blame PHPStorm.

**Comment by ncla** [comment]
Don't know what you talking about. :^)

**Comment by ncla** [comment]
Nevermind.

**Comment by ncla** [comment]
Check now.

**Comment by xPaw** [comment]
Thanks!

**Pull request merged**